docs/TRANSPARENCY_FAQ.md

```diff
@@ -0,0 +1,27 @@
+## Chat with your data Solution Accelerator: Responsible AI FAQ
```

Comment:
Can we lowercase the filename? 

Also wondering if `responsible_ai_faq.md` would be a better name?

Reply:
we have to leave the name as is because it is a CELA approval, but I can make it lowercase.